metadata_service/proxy/__init__.py

```diff
@@ -0,0 +1,37 @@
+from threading import Lock
+
+from flask import current_app
+from typing import Type
+from werkzeug.utils import import_string
+
+from metadata_service import config
+from metadata_service.proxy.base_proxy import BaseProxy
+
+_proxy_client = None
+_proxy_client_lock = Lock()
+
+
+def get_proxy_client() -> Type[BaseProxy]:
```

Comment:
@verdan Could you replace `Type[BaseProxy]` to `BaseProxy`? I found mypy error is gone after this

Reply:
nice catch!! I used the `Type[]` in the wrong way here. 👍